src/agents/utility.jl

```diff
@@ -1506,6 +1533,12 @@ function solve_agent_problem!(
                     customers.x_DG_new_my(Symbol(Int(y_symbol)), h, z, m) for y_symbol in
                     model_data.year(first(model_data.index_y_fix)):model_data.year(y)
                 ) for h in model_data.index_h, m in customers.index_m
+            ) + 
+            sum(
```

Comment:
Missing comment? How is this different from `# remove aggregated behind-the-meter storage generation/consumption ...`?